lite-pm/pages/components/AppPage.js

```diff
@@ -0,0 +1,151 @@
+import React, { Component } from "react";
+import {
+  Container,
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+} from "reactstrap";
+import Navigationbar from "./Navigationbar";
+import ProjectDetails from "./ProjectDetails";
+import styles from "../../styles/AppPage.module.css";
+import TeamMember from "./TeamMember";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+
+class AppPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      teamMembers: [],
+      showModal: false,
+      memberName: "Name",
+      memberEmail: "Email",
+      memberGit: "Github Link",
+      memberPhone: "Phone Number",
+    };
+    this.addTeamMember = this.addTeamMember.bind(this);
+    this.toggleAddMemberModal = this.toggleAddMemberModal.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.renderCreateMemberModal = this.renderCreateMemberModal.bind(this);
```

Comment:
No need to change, but you can also modify the definition of the function to bind it automatically, e.g. `this.renderCreateMemberModal = () => { your code }`